debloat.sh

```diff
@@ -1,27 +1,23 @@
 #!/bin/sh
-package_file=packages.txt
+
 logfile=adb.log
-start=$(date +%s.%N)
-packages=$(cat $package_file)
-lines=$(wc -l < $package_file)
-removed=0
+start_time=`date +%s`
```

Comment:
This was already working and correct.
https://github.com/koalaman/shellcheck/wiki/SC2006